applications/GeoMechanicsApplication/custom_processes/apply_excavation_process.hpp

```diff
@@ -10,61 +10,32 @@
 //  Main authors:    Lorenzo Gracia,
 //                   Aron Noordam,
 //                   Vahid Galavi
+//                   Marjan Fathian
 //
 #pragma once
 
-// Project includes
-#include "includes/model_part.h"
-#include "includes/element.h"
-#include "utilities/variable_utils.h"
-
-// Application includes
-#include "includes/kratos_flags.h"
-#include "includes/kratos_parameters.h"
 #include "processes/process.h"
 
 namespace Kratos
 {
 
-class ApplyExcavationProcess : public Process
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) ApplyExcavationProcess : public Process
 {
-  public:
+public:
     KRATOS_CLASS_POINTER_DEFINITION(ApplyExcavationProcess);
 
+    ApplyExcavationProcess(ModelPart&        rModelPart,
+                           const Parameters& rSettings);
```

Comment:
Since you have updated the signature of the constructor, please also update it in file `add_custom_processes_to_python.cpp` (add `const` to the `Parameters&` parameter). Search for `ApplyExcavationProcess` in that file.